programs/keeper-client/Commands.h

```diff
@@ -81,9 +87,63 @@ class GetCommand : public IKeeperClientCommand
 
     void execute(const ASTKeeperQuery * query, KeeperClient * client) const override;
 
-    String getHelpMessage() const override { return "get <path> -- Returns the node's value"; }
+    String getHelpMessage() const override { return "{} <path> -- Returns the node's value"; }
+};
+
+class GetStatCommand : public IKeeperClientCommand
+{
+    String getName() const override { return "getstat"; }
```

Comment:
```suggestion
    String getName() const override { return "get_stat"; }
```
We have `find_super_nodes`, `delete_stable_backups`, `find_big_family`, but `getstat`😕. I think it's better to use `get_stat`.